.github/workflows/CI.yml

```diff
@@ -0,0 +1,95 @@
+name: CI
+
+on:
+  push
```

Comment:
Do we want to run CI on pull requests too?
on: [push, pull_request]

Reply:
All PRs have pushes, so afaik, this should run on PRs as well. No need to run it twice. Unless there's some edge case I'm missing?

Reply:
Good points. Will make the changes.